pyproject.toml

```
@@ -0,0 +1,21 @@
+[project]
+name = "xcp-ng-build"
+description = "A tool to create XCP-ng-based build environments"
+readme = "README.md"
+requires-python = ">=3.9"
```

```suggestion
requires-python = ">=3.11"
```
to match the version in the template.
There should be the `.python-version` file too

The other conf from the template could probably wait :)

---

> There should be the .python-version file too

I've never had to use such a file and haven't seen it mentioned in any PyPA docs, can you link to it?

> To build the image, why not `build-image`?

Do you mean as a subcommand? That can be done, yes.

If you mean it as a command name, I'd rather not use such a generic name

---

`.python-version` was introduced by `pyenv` IIRC, and is now used by several other tools.
 `uv` uses it to create a default python version request https://docs.astral.sh/uv/concepts/python-versions/#python-version-files

---

I don't see the point of having it if uv can already follow the python version in pyproject.toml, and I don't like to have duplication